### ISSUE TITLE
Missing attribute file after upgrading LVM package on RHEL7

### ISSUE THREAD

**Issue by dvschwartz** [chef-cookbooks/lvm#166]
### Cookbook version
4.5.1

### Chef-client version
12.16.42

### Platform Details
RHEL7 (kernel version-locked to 7.4) on AWS EC2, managed by OpsWorks Chef Automate.

### Scenario:
Install latest OS package updates per monthly schedule

### Steps to Reproduce:
As soon as package updates are installed, the client fails during converge due to an error with "lvm_physical_volume("/dev/xvdb") do"...

### Expected Result:
Successful converge.

### Actual Result:
After the LVM package is updated from lvm2-2.02.177-4.el7.x86_64 to lvm2-2.02.180-10.el7_6.2.x86_64, the converge fails with the following error:

Unable to load lvm attributes [lvs.yaml] for version [2.02.180(2)]. The version/object may not be supported or you may need to upgrade the chef-ruby-lvm-attrib gem. Error [No such file or directory @ rb_sysopen - /opt/chef/embedded/lib/ruby/gems/2.3.0/gems/chef-ruby-lvm-attrib-0.2.5/lib/lvm/attributes/2.02.180(2)/lvs.yaml]

Upon examination of the target directory (/opt/chef/embedded/lib/ruby/gems/2.3.0/gems/chef-ruby-lvm-attrib-0.2.5/lib/lvm/attributes/), the highest version named directory is "2.02.177(2)".

**Comment by welcomebot** [comment]
Hey There
It looks like this is the first issue you've filed against the chef-cookbooks project. I'm here to offer you a bit of extra help to make sure we can quickly get back to you.
 Make sure you've filled out all the fields in our issue template. Make sure you've provided us with the version of chef-client you're running, your operating system and the version of the cookbook. If you're not using the most up to date version of the cookbook then please make sure to update first. Lots of things change between versions even if you're issue isn't listed in the changelog. Finally please give us a detailed description of the issue you're having. The more we know about what you're trying to do, what actually happens, and how you can reproduce the problem, the better.

If you're looking for more immediate troubleshooting help make sure to check out #general on the Chef Community Slack. There's plenty of folks there willing to lend a helping hand. Thanks for the first issue. We hope we can get back to you soon with a solution.

**Comment by hfridell** [comment]
I was getting a similar error try bumping the gem version
```    
"chef-ruby-lvm-attrib": {
      "version": "0.2.6"
    }

**Comment by dvschwartz** [comment]
> I was getting a similar error try bumping the gem version
> 
> ```
> "chef-ruby-lvm-attrib": {
>       "version": "0.2.6"
>     }
> ```

Thanks, @hfridell.

I manually updated the gem, as you suggested:
`# /opt/chef/embedded/bin/gem update chef-ruby-lvm-attrib`

That bumped it to 0.2.6 and created the new "2.02.180(2)" directory, along with 181 and 182.  However, when I run chef-client, it shows the exact same error.  It's still looking for lvs.yaml in the previous version directory path.

When I looked at the LVM cookbook default attributes, I saw the problem:
`default['lvm']['chef-ruby-lvm-attrib']['version'] = '0.2.5'`

I assume that I can just override the attribute in my custom cookbook and set it to 0.2.6 as a work-around, but I don't want to indefinitely test and maintain that.  What would be the correct course of action for a long-term solution?

Thanks!
